plugins/module_utils/acme/acme.py

```diff
@@ -399,8 +401,19 @@ def create_backend(module, needs_acme_v2):
 
     # Create backend object
     if backend == 'cryptography':
+        if CRYPTOGRAPHY_ERROR is not None:
+            # Either we couldn't import cryptography at all, or there was an unexpected error
+            if CRYPTOGRAPHY_VERSION is None:
+                msg = missing_required_lib('cryptography')
+            else:
+                msg = 'Unexpected error while preparing cryptography: {0}'.format(CRYPTOGRAPHY_ERROR.splitlines()[-1])
+            module.fail_json(msg=msg, exception=CRYPTOGRAPHY_ERROR)
         if not HAS_CURRENT_CRYPTOGRAPHY:
-            module.fail_json(msg=missing_required_lib('cryptography'))
+            # We could import cryptography, but an too old version.
```

Comment:
```suggestion
            # We could import cryptography, but the version might be too old.
```

Reply:
At this point we know that the version is too old, so 'might be' doesn't sound correct to me. How about:
```suggestion
            # We could import cryptography, but the version is too old.
```
?

Reply:
or even better
```suggestion
            # We can import cryptography, but the version is too old.
```

Reply:
```suggestion
            # We are able to import cryptography, but the version is too old.
```

Reply:
```suggestion
            # We succeeded importing cryptography, but its version is too old.
```